### ISSUE TITLE
Capturing not used options/values

### ISSUE THREAD

**Issue by sn50** [gsscoder/commandline#268]
Let's say I need one option `-p` as a valid option for my program. Everything else is invalid. I need to capture those invalid ones and, let's say, show them to the user. How am I about to do that?

E.g. for `program.exe -g -f file` I need to get something like `{'g'=> null, 'f'=>"file"}` in any form available. I couldn't find out how to do that.

Is it possible? If yes, how to achieve it? Thank you in advance.

**Comment by nemec** [comment]
No, I don't believe that feature is available. What use cases are you thinking of for this feature?

**Comment by sn50** [comment]
I want my simple program to be able to explicitly inform user on his invalid options entered, since I consider every unpaired option invalid and worth reporting to user.

**Comment by mizipzor** [comment]
You could use a `ValueList` to pick up all additional arguments and then check yourself if that list is empty after parsing. This is done differently depending on branch, what version are you using?
